src/views/ConsentContinuationOut.vue

```diff
@@ -735,19 +816,25 @@ export default class ConsentContinuationOut extends Vue {
     this.haveChanges = true
   }
 
+  @Watch('documentDeliveryValid')
+  onDocumentDeliveryValidChanged (): void {
+    this.haveChanges = true
+  }
+
   @Watch('certifyFormValid')
   onCertifyFormValidChanged (): void {
     this.haveChanges = true
   }
 
+  @Watch('courtOrderValid')
+  onCourtOrderValidChanged (): void {
+    this.haveChanges = true
+  }
```

Comment:
I think you can watch all 3 in a row and have a common function for them, eg,
```
watch first thin
watch second thing
watch third thing
private onHaveChanges () {
  ...
}
```

Reply:
I just tried this and it works! This is awesome, thanks Sev.